components/layout_2020/flow/mod.rs

```diff
@@ -764,7 +763,8 @@ fn layout_in_flow_non_replaced_block_level_same_formatting_context(
         (computed_min_block_size.is_definitely_zero() || computed_min_block_size.is_auto());
 
     let block_size = containing_block_for_children.block_size.auto_is(|| {
-        content_block_size
+        content_block_size;
```

Comment:
It doesn't look like this line is doing anything. It's just a variable followed by a semicolon.